pkg/pipelines/bootstrap.go

```diff
@@ -1,196 +1,199 @@
+		}
+	}
+	outputs := make([]interface{}, 0)
+	namespaces := namespaceNames(o.Prefix)
+	for _, n := range createNamespaces(values(namespaces)) {
+		outputs = append(outputs, n)
+	}
+
+	githubAuth, err := createOpaqueSecret(meta.NamespacedName(namespaces["cicd"], "github-auth"), o.GithubToken)
+	if err != nil {
+		return fmt.Errorf("failed to generate path to file: %w", err)
+	}
+	outputs = append(outputs, githubAuth)
+
+	// Create Docker Secret
+	dockerSecret, err := createDockerSecret(o.QuayAuthFileName, namespaces["cicd"])
+	if err != nil {
+		return err
+	}
+	outputs = append(outputs, dockerSecret)
+
+	// Create Tasks
+	tasks := tasks.Generate(githubAuth.GetName(), namespaces["cicd"])
+	for _, task := range tasks {
+		outputs = append(outputs, task)
+	}
+
+	// Create trigger templates
+	templates := triggers.GenerateTemplates(namespaces["cicd"], saName)
+	for _, template := range templates {
+		outputs = append(outputs, template)
+	}
+
+	// Create trigger bindings
+	bindings := triggers.GenerateBindings(namespaces["cicd"])
+	for _, binding := range bindings {
+		outputs = append(outputs, binding)
+	}
+
+	// Create Pipelines
+	outputs = append(outputs, createPipelines(namespaces, o.DeploymentPath)...)
+
+	// Create Event Listener
+	eventListener := eventlisteners.Generate(o.GitRepo, namespaces["cicd"], saName)
+	outputs = append(outputs, eventListener)
+
+	// Create route
+	route := routes.Generate(namespaces["cicd"])
+	outputs = append(outputs, route)
+
+	//  Create Service Account, Role, Role Bindings, and ClusterRole Bindings
+	outputs = append(outputs, createRoleBindings(namespaces)...)
+
+	return marshalOutputs(os.Stdout, outputs)
 }
 
 func createRoleBindings(ns map[string]string) []interface{} {
-    out := make([]interface{}, 0)
-    sa := createServiceAccount(meta.NamespacedName(ns["cicd"], saName), dockerSecretName)
-    out = append(out, sa)
-    role := createRole(meta.NamespacedName(ns["cicd"], roleName), rules)
-    out = append(out, role)
-    out = append(out, createRoleBinding(meta.NamespacedName(ns["cicd"], roleBindingName), sa, role.Kind, role.Name))
-    out = append(out, createRoleBinding(meta.NamespacedName(ns["cicd"], "edit-clusterrole-binding"), sa, "ClusterRole", "edit"))
-    return out
+	out := make([]interface{}, 0)
+	sa := createServiceAccount(meta.NamespacedName(ns["cicd"], saName), dockerSecretName)
+	out = append(out, sa)
+	role := createRole(meta.NamespacedName(ns["cicd"], roleName), rules)
+	out = append(out, role)
+	out = append(out, createRoleBinding(meta.NamespacedName(ns["cicd"], roleBindingName), sa, role.Kind, role.Name))
+	out = append(out, createRoleBinding(meta.NamespacedName(ns["cicd"], "edit-clusterrole-binding"), sa, "ClusterRole", "edit"))
+	out = append(out, createRoleBinding(meta.NamespacedName(ns["cicd"], devRoleBindingName), sa, "ClusterRole", "edit"))
+	out = append(out, createRoleBinding(meta.NamespacedName(ns["cicd"], stageRoleBindingName), sa, "ClusterRole", "edit"))
+
```

Comment:
Need to create RoleBindings in "dev" and "stage" envirionments.

Reply:
made the mentioned changes